style.css

```
@@ -0,0 +1,259 @@
+    text-decoration: none;
+}
+section .container .contactinfo .sci li a img{
+    filter: invert(1);
+    opacity: 0.5;
+}
+section .container .contactinfo .sci li:hover a img{
+    opacity: 1;
+}
+section .container .contactform{
+    position: absolute;
+    padding: 70px 50px;
+    background: #fff;
+    margin: 0 0 0 150px;
+    padding: 0 0 0 250px;
+    width: calc(100% - 150px);
+    height: 100%;
+    box-shadow: 0 50px 50px rgba(0, 0, 0, 0.5);
+}
+section .container .contactform h2{
+    color: #0f3950;
+    font-size: 24px;
+    font-weight: 500;
+}
+section .container .contactform .formbox{
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: 30px 0 0 0;
+}
+section .container .contactform .formbox .inputbox{
+    position: relative;
+    margin: 0 0 35px 0;
+}
+section .container .contactform .formbox .inputbox.w50{
+    width: 47%;
+}
+section .container .contactform .formbox .inputbox.w100{
+    width: 100%;
+}
+section .container .contactform .formbox input{
+    width: 100%;
+    padding: 5px 0;
+}
+section .container .contactform .formbox .inputbox input,
+section .container .contactform .formbox .inputbox textarea
+{
+    width: 100% !important;
+    padding: 5px;
+    resize: none;
+    font-size: 17px;
+    font-weight: 300;
+    color: #333;
+    border: none;
+    border-bottom: 1px solid #777;
+    outline: none;
+}
+section .container .contactform .formbox .inputbox textarea{
+    min-height: 120px;
+}
+section .container .contactform .formbox .inputbox span{
+    position: absolute;
+    left: 0;
+    padding: 5px 0;
+    font-size: 18px;
+    font-weight: 300;
+    color: #333;
+    transition: 0.5s;
+    pointer-events: none;
+}
+section .container .contactform .formbox .inputbox textarea:focus ~ span,
+section .container .contactform .formbox .inputbox input:focus ~ span,
+section .container .contactform .formbox .inputbox input:valid ~ span,
+section .container .contactform .formbox .inputbox textarea:valid ~ span
+{
+    transform: translateY(-20px);
+    font-size:12px;
+    font-weight: 400;
+    letter-spacing: 1px;
+    color: #ff569c;
+}
+section .container .contactform .formbox .inputbox input[type="submit"]{
+    position: relative;
+    cursor: pointer;
+    background: #0f3959;
+    color: #fff;
+    border: none;
+    max-width: 150px;
+    padding: 12px;
+}
+section .container .contactform .formbox .inputbox input[type="submit"]:hover{
+    background: #ff569c;
+}
+/* responisve */
+@media (max-width : 1200px)
+{
+    section .container{
+    width: 90%;
+    min-width: auto;
+    margin: 20px;
+    box-shadow: 0 20px 50px rgba(0, 0, 0, 0.2);
+}
+    section .container .contactinfo
+    {
+    position: relative;
+    height: 500px;
+    top: 0;
+    box-shadow: none;
+    }
+    section .container .contactform
+    {
+    width: calc(100% - 350px);
+    position: relative;    
+    margin-left: 0; 
+    padding-left: 40px;
```

```suggestion
    padding-left: 0px;
```
for the same issue change number-3